libcaf_core/caf/abstract_actor.cpp

```diff
@@ -165,8 +165,8 @@ bool abstract_actor::cleanup(error&& reason, execution_unit* host) {
   });
   if (!set_fail_state)
     return false;
-  CAF_LOG_DEBUG("cleanup" << CAF_ARG(id()) << CAF_ARG(node())
-                          << CAF_ARG(fail_state_));
+  log::core::debug("cleanup id = {} node = {} fail-state = {}", id(), node(),
```

Comment:
To make this a bit more readable:

```suggestion
  log::core::debug("cleanup: id = {}, node = {}, fail-state = {}", id(), node(),
```